src/targets/gpu/mlir.cpp

```diff
@@ -369,9 +370,9 @@ struct mlir_program
             return *this;
         }
 
-        mlir_operation_state& add_results(const std::vector<shape>& outputs)
+        mlir_operation_state& add_results(const std::vector<shape>& outputs, bool bStandard)
         {
-            auto x = prog->make_tensors(outputs);
+            auto x = prog->make_tensors(outputs, bStandard);
```

Comment:
I would rather convert the shape to standard instead of trying to pass a boolean through all of this.

Reply:
Do you mean, having something like
shape::get_standardize()
which recalculates stride for the lens, ignoring the inferred shape?

That could workaround this. little hacky.